client/src/index.jsx

```diff
@@ -44,7 +44,7 @@ volumeUp(){
   }
 
 get(){
-  $.get('/media', (data) => {
+  $.get('/media/1', (data) => {
```

Comment:
maybe add a comment clarifying that '1' is the id of our presentation example album?